src/Servers/Kestrel/perf/Microbenchmarks/Http2/Http2ConnectionBenchmarkBase.cs

```diff
@@ -56,6 +53,16 @@ public virtual void GlobalSetup()
         _httpRequestHeaders[HeaderNames.Scheme] = new StringValues("http");
         _httpRequestHeaders[HeaderNames.Authority] = new StringValues("localhost:80");
 
+        if (NumCookies > 0)
+        {
+            var cookies = new string[NumCookies];
+            for (var index = 1; index < NumCookies; index++)
```

Comment:
```suggestion
            for (var index = 0; index < NumCookies; index++)
```

Reply:
I also had this wrong when I first suggested this update.

Reply:
Sorry, that was an accidental push. I was still testing. I keep confusing the VS pull and sync button with a refresh git window button. I was just about to fix that. You are so fast haha